bundles/org.openhab.binding.ism8/src/main/java/org/openhab/binding/ism8/internal/Ism8Handler.java

```diff
@@ -0,0 +1,166 @@
+                    && channel.getConfiguration().get("write").toString().equalsIgnoreCase("true")) {
+                IDataPoint dataPoint = null;
+                try {
+                    int id = Integer.parseInt(channel.getConfiguration().get("id").toString());
+                    this.logger.debug("Channel '{}' writting into ID '{}'", channel.getUID().getId(), id);
+                    this.updateState(channelUID, new QuantityType<>(command.toString()));
+
+                    for (IDataPoint dp : svr.getDataPoints()) {
+                        if (dp.getId() == id) {
+                            dataPoint = dp;
+                            break;
+                        }
+                    }
+                } catch (NumberFormatException e) {
+                    this.logger.debug("Updating State of ISM DataPoint '{}' failed. '{}'", channel.getConfiguration(),
+                            e.getMessage());
+                }
+
+                if (dataPoint != null) {
+                    try {
+                        svr.sendData(dataPoint.createWriteData(command));
+                    } catch (IOException e) {
+                        this.logger.debug("Writting to ISM DataPoint '{}' failed. '{}'", dataPoint.getId(),
+                                e.getMessage());
+                    }
+                }
+            }
+        }
+    }
+
+    @Override
+    public void dispose() {
+        Server svr = this.server;
+        if (svr != null) {
+            svr.stopServerThread();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        this.config = getConfigAs(Ism8Configuration.class);
+        Ism8Configuration cfg = this.config;
+        if (cfg != null) {
+            Server svr = new Server(cfg.getPortNumber());
+            for (Channel channel : getThing().getChannels()) {
+                if (channel.getConfiguration().containsKey("id") && channel.getConfiguration().containsKey("type")) {
+                    try {
+                        int id = Integer.parseInt(channel.getConfiguration().get("id").toString());
+                        String type = channel.getConfiguration().get("type").toString();
+                        String description = channel.getLabel();
+                        if (type != null && description != null) {
+                            svr.addDataPoint(id, type, description);
+                        }
+                    } catch (NumberFormatException e) {
+                        this.logger.warn(
+                                "Ism8 initialize: ID couldn't be converted correctly. Check the configuration of channel {}. Cfg={}",
+                                channel.getLabel(), channel.getConfiguration());
+                    }
+                } else {
+                    this.logger.info("Ism8: ID or type missing - Channel={}  Cfg={}", channel.getLabel(),
+                            channel.getConfiguration());
```

Comment:
Logging to info should be used rarely. This could be debug.

Reply:
OK, updated.